_includes/head.html

```diff
@@ -32,16 +32,18 @@
 	<meta name="msapplication-config" content="{{ site.baseurl }}/assets/img/favicon/browserconfig.xml">
     <!--favicon-->
 
+	<!--Shared styles-->
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
     <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
 	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-	<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
 	<link rel="stylesheet" type="text/css" href="https://csshake.surge.sh/csshake.min.css">
 	<link rel="stylesheet" href="{{ site.baseurl }}/assets/css/main.css">
-	<link rel="stylesheet" href="{{ site.baseurl }}/styles.css">
```

Comment:
Haha whoops, guess it wasn't ideal to add this here, thanks :)

Reply:
Thanks for merging Rebecca! :)